### ISSUE TITLE
Can monitor the data of the project being proxied

### ISSUE THREAD

**Issue by zjw186** [xetys/microservices-sidecar-example#1]
If i use sidecar to proxy the project, can I do health check, not only the service start or not?

**Comment by xetys** [comment]
yes you can, but the application must provide it's own healtcheck endpoints on "/health"
